src/main/java/frc/robot/Robot.java

```diff
@@ -157,45 +157,35 @@ private void configureBindings() {
         .driverController
         .rightTrigger()
         .onTrue(robotCommands.confirmShotCommand())
+        .onTrue(robotCommands.waitSpeakerCommand())
```

Comment:
make sure you only have one onTrue/onFalse per button. otherwise multiple requests will get triggered at the same time which is probably undesirable